.circleci/config.yml

```diff
@@ -426,7 +427,111 @@ jobs:
           comment: "Failed to Clean up prnet"
           pr_number: << pipeline.parameters.pr_number >>
 
-  build-and-push-docker-image:
+  build-helm-chart:
+    environment:
+      - IMAGE: taraxa-node
+      - HELMCHART: taraxa-node
+    machine:
+      image: ubuntu-2204:2022.04.2
+      docker_layer_caching: true
+    resource_class: medium
+    steps:
+      - checkout
+
+      - run:
+          name: Prepare Environment
+          command: |
+            echo "export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )" >> $BASH_ENV
+            echo "export SHORT_GIT_HASH=$(echo $CIRCLE_SHA1 | cut -c -7)" >> $BASH_ENV
+            export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )
+            echo "export HELM_TEST_NAME=pr-${PR}" >> $BASH_ENV
+            curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
+            sudo apt-get install apt-transport-https --yes
+            echo "deb https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+            sudo apt-get update
+            sudo apt-get install helm
+            helm plugin install https://github.com/chartmuseum/helm-push.git
+            helm repo add --username ${HELM_REPO_USERNAME} --password ${HELM_REPO_PASSWORD} taraxacharts https://charts.gcp.taraxa.io
+            helm repo add bitnami https://charts.bitnami.com/bitnami
+            helm repo update
+            mkdir -p $HOME/.kube
+            echo -n "${KUBE_CONFIG}" | base64 --decode > $HOME/.kube/config
+            curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+            sudo cp kubectl /usr/local/bin 
+            sudo chmod +x /usr/local/bin/kubectl
+            sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
+            sudo chmod a+x /usr/local/bin/yq
+
+      - run:
+          name: Verify Helm chart with linter
+          command: |
+            if [[ ${PR} != "false" ]]; then
+              helm lint charts/${HELMCHART}/
```

Comment:
Why not use the new orb lint functionality?

Reply:
Refactored. Using now orb. Look comment: https://github.com/Taraxa-project/taraxa-node/pull/2110/files/b7f7a7d322e7a8db10f37ba638067dab897ebd00#r992369570